fili-core/src/main/java/com/yahoo/bard/webservice/table/BaseSchema.java

```diff
@@ -29,6 +30,13 @@ protected BaseSchema(Granularity granularity, Iterable<Column> columns) {
     }
 
     @Override
+    public Optional<Column> getColumn(String columnName) {
+        return getColumns().stream()
+                .filter(column -> column.getName().equals(columnName))
+                .findFirst();
+    }
```

Comment:
I will throw an exception to it, since we only support druid right now so it should be fine for now. If not at least we hit an exception.